include/onnxruntime/core/providers/tensorrt/tensorrt_provider_factory.h

```diff
@@ -9,6 +9,37 @@ extern "C" {
 
 ORT_API_STATUS(OrtSessionOptionsAppendExecutionProvider_Tensorrt, _In_ OrtSessionOptions* options, int device_id);
 
+/**
+ * Use this API to create the configuration of a TensorRT Execution Provider.
+ * \param out - pointer to the pointer of TensorRT EP provider options instance.
+ */
+ORT_API_STATUS(OrtCreateTensorRTProviderOptions, _Outptr_ OrtTensorRTProviderOptions** out);
```

Comment:
This should all be in the C API, we shouldn't be exporting more flat functions we can't version.

Reply:
Moved these functions in the C API.